portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,41 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    ArrayList<String> messages = new ArrayList<String>();
+
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Katherine!</h1>");
+
+    messages.add("Hello and welcome to my portfolio!");
+    messages.add("Bonjour et bienvenue dans mon portfolio!");
+    messages.add("Hola y bienvenidas a mi portafolio!");
+
+    response.setContentType("application/json;");
+    String json = convertToJSON(messages);
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
+   */
+  private String convertToJSON(ArrayList<String> messages) {
+    String json = "{";
```

Comment:
What do you think about using StringBuilder? https://stackoverflow.com/questions/5234147/why-stringbuilder-when-there-is-string

Reply:
I tried and it somehow broke the code. The buttons weren't outputting anything anymore so I went back to manually appending my strings. Maybe I did something wrong but I couldn't figure out what.

Reply:
No prob! Glad you took a look at it!